flexget/components/sites/sites/site_rutracker.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+
+from loguru import logger
+from requests.exceptions import RequestException
+
+from flexget import plugin
+from flexget.components.sites.urlrewriting import UrlRewritingError
+from flexget.event import event
+
+from urllib.parse import urlparse, parse_qs
+
+
+logger = logger.bind(name='rutracker')
+
+
+class SiteRutracker:
+    schema = {'type': 'boolean'}
+
+    base_url = 'https://api.t-ru.org'
+
+    # urlrewriter API
+    def url_rewritable(self, task, entry):
+        url = entry['url']
+        if url.startswith('https://rutracker.org/forum/viewtopic.php?t='):
+            return True
+        return False
```

Comment:
more pythonic to do this:
```suggestion
        return url.startswith('https://rutracker.org/forum/viewtopic.php?t=')
```